Pressing the "Mods" button causes the game to crash

**Platform**
Operating system type, major version, architecture (64/32 bit).
- Windows 10 Build 2004, 64-bit

**Version**
Where you got the game and the exact version number.
- 3.16.13, from this GitHub repository

**Describe the bug**
A clear and concise description of what the bug is.
- When trying to press the "Mods" button to access the mod list, the game crashes, regardless of whether the mods folder is empty, not empty, or nonexistent.

**To Reproduce**
Steps to reproduce the behavior:
1. Press the "Mods" button
2. Crash happens

If game data is required, please add it here. From the menu, choose save game (or map), copy to clipboard, and paste here between the backtick blocks (or, if no such save is required, you may delete this):
- None

**Expected behavior**
A clear and concise description of what you expected to happen.
- It *should* display a list of mods that can be downloaded.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

Friggin' nasty to repro. I dug out a medieval Java 8 201 to even get console logging, the built-in eats all helpful output.

<details><summary>dump</summary>

```
Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: java.lang.ExceptionInInitializerError
        at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:134)
Caused by: java.lang.ExceptionInInitializerError
        at com.unciv.MainMenuScreen$modsTable$1.invoke(MainMenuScreen.kt:126)
        at com.unciv.MainMenuScreen$modsTable$1.invoke(MainMenuScreen.kt:125)
        at com.unciv.ui.utils.ExtensionFunctionsKt$onClick$1.invoke(ExtensionFunctions.kt:64)
        at com.unciv.ui.utils.ExtensionFunctionsKt$onClick$1.invoke(ExtensionFunctions.kt:64)
        at com.unciv.ui.utils.ExtensionFunctionsKt$onClickEvent$1.clicked(ExtensionFunctions.kt:57)
        at com.badlogic.gdx.scenes.scene2d.utils.ClickListener.touchUp(ClickListener.java:88)
        at com.badlogic.gdx.scenes.scene2d.InputListener.handle(InputListener.java:71)
        at com.badlogic.gdx.scenes.scene2d.Stage.touchUp(Stage.java:355)
        at com.badlogic.gdx.InputEventQueue.drain(InputEventQueue.java:70)
        at com.badlogic.gdx.backends.lwjgl.DefaultLwjglInput.processEvents(DefaultLwjglInput.java:255)
        at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:220)
        at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:127)
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Error reading file: jsons\ManuallyBlockedMods.json (Absolute)
        at com.badlogic.gdx.files.FileHandle.read(FileHandle.java:150)
        at com.badlogic.gdx.files.FileHandle.readString(FileHandle.java:210)
        at com.unciv.JsonParser.getFromJson(JsonParser.kt:14)
        at com.unciv.ui.pickerscreens.ModManagementScreen.<clinit>(ModManagementScreen.kt:514)
        ... 12 more
Caused by: java.io.FileNotFoundException: jsons\ManuallyBlockedMods.json (Le système ne trouve pas ce fichier)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(Unknown Source)
        at java.io.FileInputStream.<init>(Unknown Source)
        at com.badlogic.gdx.files.FileHandle.read(FileHandle.java:146)
        ... 15 more
Exception in thread "Sound" com.badlogic.gdx.utils.GdxRuntimeException: Error creating sound com.badlogic.gdx.backends.lwjgl.audio.Mp3$Sound for file: sounds/click.mp3
        at com.badlogic.gdx.backends.lwjgl.audio.OpenALLwjglAudio.newSound(OpenALLwjglAudio.java:122)
        at com.badlogic.gdx.backends.lwjgl.audio.OpenALLwjglAudio.newSound(OpenALLwjglAudio.java:41)
        at com.unciv.ui.utils.Sounds.get(Sounds.kt:143)
        at com.unciv.ui.utils.Sounds.play(Sounds.kt:163)
        at com.unciv.ui.utils.ExtensionFunctionsKt$onClickEvent$1$clicked$1.invoke(ExtensionFunctions.kt:56)
        at com.unciv.ui.utils.ExtensionFunctionsKt$onClickEvent$1$clicked$1.invoke(ExtensionFunctions.kt:56)
        at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30)
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at com.badlogic.gdx.backends.lwjgl.audio.OpenALLwjglAudio.newSound(OpenALLwjglAudio.java:120)
        ... 6 more
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Error reading audio data.
        at com.badlogic.gdx.backends.lwjgl.audio.Mp3$Sound.<init>(Mp3.java:138)
        ... 11 more
Caused by: java.lang.UnsatisfiedLinkError: org.lwjgl.openal.AL10.nalBufferData(IIJII)V
        at org.lwjgl.openal.AL10.nalBufferData(Native Method)
        at org.lwjgl.openal.AL10.alBufferData(AL10.java:1161)
        at com.badlogic.gdx.backends.lwjgl.audio.OpenALSound.setup(OpenALSound.java:52)
        at com.badlogic.gdx.backends.lwjgl.audio.Mp3$Sound.<init>(Mp3.java:136)
        ... 11 more
```
</details>

... still analyzing. Looks like an internal/external mismatch, but better look thoroughly.

This part looks interesting:
```
Caused by: java.io.FileNotFoundException: jsons\ManuallyBlockedMods.json (Le système ne trouve pas ce fichier)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(Unknown Source)
        at java.io.FileInputStream.<init>(Unknown Source)
        at com.badlogic.gdx.files.FileHandle.read(FileHandle.java:146)
```

--- a/core/src/com/unciv/ui/pickerscreens/ModManagementScreen.kt
+++ b/core/src/com/unciv/ui/pickerscreens/ModManagementScreen.kt
@@ -558,7 +558,7 @@ class ModManagementScreen: PickerScreen(disableScroll = true) {
 
     companion object {
         val modsToHideAsUrl = run {
-            val blockedModsFile = FileHandle("jsons/ManuallyBlockedMods.json")
+            val blockedModsFile = Gdx.files.internal("jsons/ManuallyBlockedMods.json")
             JsonParser().getFromJson(Array<String>::class.java, blockedModsFile)
         }
         val modsToHideNames = run {

